english/luke-skywalker/luke-skywalker.md

```diff
@@ -0,0 +1,8 @@
+So you have an interview with Luke Skywalker.
+What do you ask him?
+
+[Are you Rey's Father?](starwars8/wait-for-8.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.